changelogs/2.11.6.md

```diff
@@ -0,0 +1,158 @@
+# 2.11.6
+
+Date: 2025-02-24
+Tag: 2.11.6
+
+## Announcement: `double` field type problems
+
+Users who ever used the `double` field type in vinyl indexes might have their
+indexes broken. Regardless of this release. Preferably prior to the upgrade the
+users must get rid of the `double` indexed field type in vinyl. It must be
+drop-in replaceable by the `number` field type.
+
+The reason is that the sorting of `double` fields would always compare the field
+values as C-language doubles, even when actual integers are stored. This works
+fine for small numbers, but starting from 2^53 the double-style comparison
+looses precision. For example, 18446744073709551615 and 18446744073709551614
+would be considered the same values.
+
+Besides, point-lookups in such index are going to be broken by design too -
+looking up by 18446744073709551615 can return 18446744073709551614 and many
+other integers around this point.
+
+Unfortunately, even if the user doesn't have the `double` type in any vinyl
+indexes, but **ever had it before**, it is still unsafe. A `double` index
+altered to `number` or `scalar` would remain broken. It is **very important that
+the users rebuild all the vinyl indexes that have or ever had the `double` field
```

Comment:
```suggestion
users rebuild all the vinyl indexes that have or ever had the `double` field
```

Reply:
Fixed.

Reply:
Sent for `3.4.0`, `3.3.2`, `3.2.2` in #11196.